### ISSUE TITLE
Returning block that lives on the local stack

### ISSUE THREAD

**Issue by uestchang** [Masonry/Masonry#561]
### New Issue Checklist

🚫 If this template is not filled out your issue **will** be closed with no comment. 🚫

* [Yes ] I have looked at the [Documentation](https://github.com/SnapKit/Masonry/blob/master/README.md)
* [Yes] I have filled out this issue template.

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform                | e.g. ios
 Platform Version        | e.g. 8.0
 Masonry Version         | e.g. 1.0.1
 Integration Method      | e.g. manually
 

### Issue Description

I try to use Masonry in my project ,However , there are some errors when i Build ,
and the error is:
`/Users/.../Masonry/View+MASAdditions.m:85:12: Returning block that lives on the local stack`
My Xcode version is 10.0 ,can any guys help me?

**Comment by Ververtom** [comment]
Yes, I dragged the file to my project, then after I compiled the project, it had an error report "Returning block that lives on the local stack" in MASCompositeConstraint.m in line 52   `return ^id(CGFloat multiplier) `  
I finally found that it's related to ARC,if I set Objective-C Automatic Reference Counting NO, this error appear
